www/css/ios-overrides.css

```diff
@@ -90,7 +90,10 @@
 }
 
 .user-content {
-  background: linear-gradient(to bottom,#eb7125 60%, #efeff4 61%);
+  /*background: linear-gradient(to bottom,#eb7125 60%, #efeff4 61%);*/
```

Comment:
Do we not want the gradient anymore?

Reply:
Nope, that doesn't do anything with the background image on top. Looks the same with and without on my emulator ^^